sources/web/datalab/config/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "consoleLogging": true,
+  "consoleLogLevel": 'warn',
```

Comment:
Can we add a comment in the run scripts as an example how to set this to verbose?